cpp/util/openssl_scoped_types.h

```diff
@@ -44,6 +44,7 @@
 #include <openssl/x509.h>
 #include <openssl/x509v3.h>
 
+namespace util {
```

Comment:
Exactly. Then, all the things that are inside a `namespace cert_trans` won't need the qualification (for one example, the `Signer` class, but I'm sure most others as well). This might possibly reduce the number of affected files almost all the way to 2-3!